### PR TITLE
re #123 Test task

### PR DESCRIPTION
**Pull request by frankie87**
- a new field added to an ErrorViewModel

### CODE REVIEW

File: src/Web/Models/ErrorViewModel.cs [hunk 1]
```diff
@@ -6,6 +6,8 @@ public class ErrorViewModel
     {
         public string RequestId { get; set; }
 
+        public bool IsSuccess { get; set; }
```
Comment (zindify) [inline src/Web/Models/ErrorViewModel.cs:9]: Shouldn't be IsSuccessful?

Comment (frankie87) [reply src/Web/Models/ErrorViewModel.cs:9]: We can use the shorter version :)))

Comment (frankie87) [reply src/Web/Models/ErrorViewModel.cs:9]: But I will fix it

Comment (zindify) [reply src/Web/Models/ErrorViewModel.cs:9]: all right, it is fine. Let's go for shorter version

**Comment by zindify** [comment]
Good to go, Elena!

**Pull request merged**